### ISSUE TITLE
Table / Missing column labels in mobile-card view with header slot.

### ISSUE THREAD

**Issue by finnkk** [buefy/buefy#791]
### Overview of the problem

Missing column labels in mobile-card view with header slot.

**Buefy** version: 0.6.5
**Vuejs** version: 2.5.16
**OS/Browser**: macOS 10.13.4 / Chrome 65.xx

### Description

Column label are missing on mobile-card view when using header slot for custom headers along with column slots not using field/label attributes.

### Steps to reproduce

- Create table with header slot
- In column slot add `<b-table-column>` for every column
- View the mobile-card view
> https://codepen.io/bonmedi/pen/KRmVqr

### Expected behavior

Columns defined in header slot should show up also in mobile-card view.

### Actual behavior

No column labels in mobile-card view.

**Comment by jtommy** [comment]
@finnkk Mobile-card needs `label` prop for each `b-table-column` but in your example you shouldn't define `default` slot.

**Comment by finnkk** [comment]
Thanks @jtommy. So there is no way to have both - a custom header and custom columns (via `b-table-column`) with working Mobile-cards? Because that would be my use case even the given  Codepen example did not really made sensible use of the custom columns.

**Comment by jtommy** [comment]
Yes but you have to remove `columns` prop to `b-table` at the moment. For example: https://codepen.io/anon/pen/JvNKoB
